### DISCUSSION TITLE
发现一个免费的gpt4网站，https://ora.sh/openai/gpt4

### DISCUSSION THREAD

**Discussion by q2408808** [viopsa233/AI-testers-Forum#73] [交流]
ora以前的gpt4是有次数限制的，现在点进去也显示一天使用一次。但是奇怪的是我刚刚登录它，可以无限正常聊天，基本能确定对话的模型是gpt4，不过只有4K的token，你们试试。

**Comment by PoseidonLi0514** [comment]
唯一奇怪的就是，他不是一个一个发，而是一口气堆好了才输出，很难受

**Reply by q2408808** [reply]
可能是中转的问题。很早以前我进这个网站，还有过问题问的多了直接卡死的情况。你用起来是不是也没有次数限制？这个网站不知道出啥问题了，这都能免费用。
